Documentation/install/upgrade.rst

```diff
@@ -598,6 +600,7 @@ The following metrics have been renamed:
 * ``cilium_operator_eni_resync_total`` to ``cilium_operator_ipam_resync_total``
 * ``cilium_operator_eni_aws_api_duration_seconds`` to ``cilium_operator_ipam_api_duration_seconds``
 * ``cilium_operator_eni_ec2_rate_limit_duration_seconds`` to ``cilium_operator_ipam_api_rate_limit_duration_seconds``
+* ``cilium_operator_ipam_ec2_resync`` to ``cilium_operator_ipam_resync``
```

Comment:
I think this line sneaked in by mistake in your backport for #12453. The rename comes from  #12413 which has not been backported as far as I can tell.

Reply:
Good catch! Fixed.